### PR TITLE
jvm-mon 0.3 (new formula)

### PR DESCRIPTION
**Pull request by ajermakovics**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adds a new formula for [jvm-mon](https://github.com/ajermakovics/jvm-mon) - Console based JVM monitoring tool

### CODE REVIEW

File: Formula/jvm-mon.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on :java => "1.8+"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"bin"/"jvm-mon"
+  end
+
+  test do
+    system "false"
```
Comment (fxcoudert) [inline Formula/jvm-mon.rb]: Nope :(
We'll need a test.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/jvm-mon.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/ajermakovics/jvm-mon"
+  url "https://github.com/ajermakovics/jvm-mon/releases/download/0.3/jvm-mon-0.3.tar.gz"
+  sha256 "9b5dd3d280cb52b6e2a9a491451da2ee41c65c770002adadb61b02aa6690c940"
+
+  depends_on :java => "1.8+"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"bin"/"jvm-mon"
```
Comment (bfontaine) [inline Formula/jvm-mon.rb]: You can use `libexec/"bin/jvm-mon"`

File: Formula/jvm-mon.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+class JvmMon < Formula
+  desc "Console based JVM monitoring"
```
Comment (bfontaine) [inline Formula/jvm-mon.rb]: "Console based" -> "Console-based"

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
```
==> echo q | /usr/local/Cellar/jvm-mon/0.3/bin/jvm-mon
Exception in thread "main" java.lang.ExceptionInInitializerError
Caused by: java.lang.IllegalStateException: J2V8 native library not loaded
	at com.eclipsesource.v8.V8.checkNativeLibraryLoaded(V8.java:195)
	at com.eclipsesource.v8.V8.createV8Runtime(V8.java:149)
	at com.eclipsesource.v8.V8.createV8Runtime(V8.java:125)
	at com.eclipsesource.v8.NodeJS.createNodeJS(NodeJS.java:58)
	at com.eclipsesource.v8.NodeJS.createNodeJS(NodeJS.java:45)
	at JvmMon.<clinit>(JvmMon.java:18)
Caused by: java.lang.UnsatisfiedLinkError: Could not load J2V8 library. Reasons: 
	no j2v8_macosx_x86_64 in java.library.path

	at com.eclipsesource.v8.LibraryLoader.loadLibrary(LibraryLoader.java:75)
	at com.eclipsesource.v8.V8.load(V8.java:71)
	at com.eclipsesource.v8.V8.createV8Runtime(V8.java:145)
	... 4 more
Error: jvm-mon: failed
```

File: Formula/jvm-mon.rb (hunk 3 above)
Comment (ajermakovics) [inline Formula/jvm-mon.rb]: Thanks! Fixed

File: Formula/jvm-mon.rb (hunk 1 above)
Comment (ajermakovics) [inline Formula/jvm-mon.rb]: Fair enough:) Added

**Comment by ajermakovics** [comment]
@fxcoudert Thanks! Looks like some path issue although the same command works locally. Any way to run this locally for debugging? Tried `brew test jvm-mon --verbose` but that also works

**Review by bfontaine — APPROVED** [review]

**Comment by ajermakovics** [comment]
@fxcoudert Fixed

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @ajermakovics for your contribution to Homebrew!

**Pull request merged**